ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/baseActivity.component.ts

```diff
@@ -238,7 +238,6 @@ export abstract class BaseActivityComponent implements OnChanges, OnDestroy {
 
     protected resetValidationState(): void {
         this.setFlags(false);
-        this.submitAttempted.next(false);
```

Comment:
This was triggering the submit announce, so the announcement was happening twice, including when going backwards. Just not needed and totally unrelated to "resetting validation state" anyways.